docs/data/material/components/snackbars/snackbars.md

```diff
@@ -141,8 +141,10 @@ The Snackbar component is composed of a root `<div>` that houses interior elemen
 </div>
 ```
 
-## Experimental APIs
+## Toolpad
```

Comment:
This was more or less right, this API will move from Toolpad to Base UI, in the next few months.

```suggestion
## Experimental APIs - Toolpad
```